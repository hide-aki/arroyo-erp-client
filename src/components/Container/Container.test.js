// Container.test.js

import React from 'react';
import Enzyme, {mount} from 'enzyme';
import toJSON from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

import {Container} from './';

Enzyme.configure({adapter: new Adapter()});

describe('<Container />', () => {
  let wrapper;
  beforeAll(() => {
    wrapper = mount(
      <Container fluid="true" fixed="true">
        Test
      </Container>
    );
  });

  it('Renders properly', () => {
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
  it('It has fluid prop', () => {
    expect(wrapper.find({fluid: true}));
  });
  it('It has fixed prop', () => {
    expect(wrapper.find({fluid: true}));
  });

  it('It renders some text passed by children', () => {
    expect(wrapper.find('Test'));
  });
});
