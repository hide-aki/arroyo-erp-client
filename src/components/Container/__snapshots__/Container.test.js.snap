// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Container /> Renders properly 1`] = `
<Memo(Container)
  fixed="true"
  fluid="true"
>
  <div
    className="container container-fluid"
    fixed="true"
    fluid="true"
  >
    Test
  </div>
</Memo(Container)>
`;
